.travis.yml

```diff
@@ -27,8 +27,8 @@ matrix:
   include:
     - node_js: "node"
       env: JOB=lint
-  # We test the latest version on circleci
-    - node_js: "11"
+  # We test the Node.js 12 on circleci
```

Comment:
Why not run 13 on Circle and make this 12?

Reply:
Because `circleci/node:13` is [not available](https://circleci.com/docs/2.0/circleci-images/#nodejs) yet. After it is available we could swap 13 to circleci.

Reply:
Gotcha, fwiw, I'm okay with waiting :)